bpf/process/argfilter_maps.h

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright Authors of Cilium */
+
+#ifndef ARGFILTER_MAPS_H__
+#define ARGFILTER_MAPS_H__
+
+#define ARGFILTER_MAPS_OUTER_MAX_ENTRIES 8
```

Comment:
Question: What means to have 8 entries in the outer map? I saw another comment that means 64 bits but not sure I follow that. Why we need 1 outer map with 8 entries and each inner map has 1 element? Why not use one BPF_MAP_TYPE_HASH with 8 entries?

Based on these, I believe that I didn't get the details on how `op_filter_inmap` works. I understand that it will remove the limit of 4(?) values that we have but not sure how.

I propose to just add some example yaml files in `crds/examples/` that use `op_filter_inmap` and with some comments there. This will make much easier to understand on the high level idea of this mechanism.

Reply:
> What means to have 8 entries in the outer map? 

 That we can have 8 tables, that is 8 different set of values to check arguments against.
 
 > Why we need 1 outer map with 8 entries and each inner map has 1 element?
 
 The 1 element max_entry will be overridden by the loader. The inner map will have a maximum size that can fit all the entries we need.
 
 See the `populateArgFilterMap` function:
 
 ```
 innerSpec := &ebpf.MapSpec{
		Name:       innerName,
		Type:       ebpf.Hash,
		KeySize:    8, // NB: hardcoded to 64 bits for now
		ValueSize:  uint32(1),
		MaxEntries: uint32(len(innerData)),
}
```